### PR TITLE
Update Skoda

### LINKED ISSUE #6399: Updating icon sources
### Brand Name

Several

### Official Resources for Icon and Color

Several.
Many icons reference svg, png, zip or pdf files as sources outside Github. Other reference uploaded files in wikipedia instead of the commons page in wikimedia, etc. The goal of this review and others to come is to standarize the source references according to the contribution guidelines, as discussed in #6364 

Sources need to change for the following icons, to the following values:

**Jamstack** : https://github.com/jamstack/jamstack.org/blob/main/src/site/img/logo/svg/Jamstack_Icon_Original.svg
**Kaufland** : https://commons.wikimedia.org/wiki/File:Kaufland_201x_logo.svg
**Open Containers Initiative** : https://github.com/opencontainers/artwork/blob/master/oci/icon/black/oci-icon-black.svg
**JavaScript** : https://github.com/voodootikigod/logo.js/blob/master/js.png
**Nrwl** : https://nrwl.io/
**Angular** : https://angular.io/presskit
**Vue.js** : https://github.com/vuejs/art/blob/master/logo.svg
**tmux** : https://github.com/tmux/tmux/blob/f04cc3997629823f0e304d4e4184e2ec93c703f0/logo/tmux-logomark.svg
**WebAuthn** : https://github.com/webauthn-open-source/webauthn-logos/blob/master/final-webauthn-logo-logo-black.svg
**Reason** : https://reasonml.github.io/
**Xamarin** : https://github.com/dotnet-foundation/swag/blob/main/xamarin/Xamarin/svg/Xamarin%20logo.svg
**Kotlin** : https://kotlinlang.org/docs/faq.html#where-can-i-get-an-hd-kotlin-logo
**Sega** : https://en.wikipedia.org/wiki/File:SEGA_logo.svg
**Windows 95** : https://en.wikipedia.org/wiki/File:Windows_Logo_(1992-2001).svg
**Redux** : https://github.com/reduxjs/redux/blob/master/logo/logo.svg
**ŠKODA** : https://commons.wikimedia.org/wiki/File:Skoda_-1200x1200.jpg
**PyPy** : https://www.pypy.org
**Swagger** : https://swagger.io/
**XMPP** : https://github.com/xsf/xmpp.org/blob/00c49187e353c1a156c95562dafaf129e688fbad/content/icons/safari-pinned-tab.svg
**Tencent QQ** : https://en.wikipedia.org/wiki/File:Tencent_QQ.svg
**X.Org** : https://en.wikipedia.org/wiki/File:X.Org_Logo.svg
**Thymeleaf** : https://github.com/thymeleaf/thymeleaf-dist/blob/main/src/site/images/thymeleaf.png
**Zig** : https://github.com/ziglang/logo/blob/master/zig-mark.svg
**InVision** : https://in.invisionapp.com/boards/FH3LW3S7XSD/
**Yamaha Motor Corporation** : https://en.wikipedia.org/wiki/File:YamahaMotorCo.svg
**Tokyo Metro** : https://en.wikipedia.org/wiki/File:Tokyo_Metro_combined_logo.svg
**Noun Project** : https://thenounproject.com/
**Treyarch** : https://en.wikipedia.org/wiki/File:Treyarch_logo.svg
**Triller** : https://triller.co/
**SketchUp** : https://www.sketchup.com/
**Society6** : https://blog.society6.com
**ResearchGate**: http://biblioteca.uoc.edu/es/recursos/recurso/researchgate																											
**MetaFilter**: https://www.metafilter.com/search.mefi?site=mefi&q=logo																											
**Alipay**: https://intl.alipay.com/																											
**Iata**: https://commons.wikimedia.org/wiki/File:IATAlogo.svg																											
**DLNA**: https://commons.wikimedia.org/wiki/File:DLNA_logo.svg																											
**Data Version Control**: https://static.iterative.ai/logo/																											
**AniList**: https://anilist.co/																											
**Google Calendar**: https://about.google/brand-resource-center/logos-list/																											
**Google Hangouts**: https://about.google/brand-resource-center/logos-list/																											
**Google Keep**: https://about.google/brand-resource-center/logos-list/																											
**Google Maps**: https://about.google/brand-resource-center/logos-list/																											
**Google Meet**: https://about.google/brand-resource-center/logos-list/																											
**Google Cardboard**: https://arvr.google.com/cardboard/																											
**Fila**: https://commons.wikimedia.org/wiki/File:Fila_logo.svg#/media/File:Fila_logo.svg																											
**XAML**: https://microsoft.github.io/microsoft-ui-xaml/																											
**NVIDIA**: https://www.nvidia.com/en-us/																											
**Arduino**: https://www.arduino.cc/																											
**Broadcom**: https://en.wikipedia.org/wiki/File:Broadcom_Ltd_Logo.svg																											
**Das Erste**: https://en.wikipedia.org/wiki/File:Das_Erste_2014.svg																											
**Facepunch**: https://facepunch.com																											
**Gmail**: https://about.google/brand-resource-center/logos-list/																											
**GNU IceCat**: https://git.savannah.gnu.org/cgit/gnuzilla.git/plain/artwork																											
**C Sharp**: https://commons.wikimedia.org/wiki/File:C_Sharp_wordmark.svg																											
**Apache Cassandra**: https://commons.wikimedia.org/wiki/File:Cassandra_logo.svg																											
**ARAL**: https://commons.wikimedia.org/wiki/File:Aral_Logo.svg																											
**OBS Studio**: https://commons.wikimedia.org/wiki/File:OBS.svg																											
**Fujifilm**: https://commons.wikimedia.org/wiki/File:Fujifilm_logo.svg																											
**São Paulo Metro**: https://commons.wikimedia.org/wiki/File:Sao_Paulo_Metro_Logo.svg																											
**iTunes**: https://commons.wikimedia.org/wiki/File:ITunes_logo.svg		
**Hack The Box**: https://www.hackthebox.eu/																											
**Robot Framework**: https://github.com/robotframework/visual-identity/blob/master/logo/robot-framework.svg																											
**Wipro**: https://www.wipro.com/																											
**TensorFlow**: https://about.google/brand-resource-center/logos-list/																									
**Aerospike**: https://aerospike.com/search/?addsearch=logo																											
**San Francisco Municipal Railway**: https://commons.wikimedia.org/wiki/File:Muni_worm_logo.svg																											
**Rakuten**: https://global.rakuten.com/corp/news/media/																											
**Safari**: https://es.wikipedia.org/wiki/Archivo:Safari_browser_logo.svg																											
**Chevrolet**: https://media.chevrolet.com/media/us/en/chevrolet/photos.detail.html/content/Pages/galleries/us/en/vehicles/chevrolet/logos.html																											
**Reliance Industries Limited**: https://en.wikipedia.org/wiki/File:Reliance_Industries_Logo.svg																																																		
**Conda-Forge**: https://github.com/conda-forge/conda-forge.github.io/blob/master/img/anvil_black.png																											
**Redbubble**: https://www.redbubble.com/																											
**OpenVPN**: https://commons.wikimedia.org/wiki/File:OpenVPN_logo.svg																											
**Citroën**: https://www.groupe-psa.com/en/brands-and-services/citroen/																																														
**Scrum Alliance**: https://scrum.mx/advanced-certified-scrum-master	
**Bose**: https://developer.bose.com/	
**Roblox**: https://corp.roblox.com/press-kit/",																						
**Jamboard**: https://workspace.google.com/products/jamboard/	
**TYPO3**: https://typo3.org/project/brand/style-guide/	
**Veritas**: https://www.veritas.com/	
**Nutanix**: https://www.nutanix.com/	
**Internet Explorer**: https://commons.wikimedia.org/wiki/File:Internet_Explorer_10%2B11_logo.svg	
**OCaml**: https://ocaml.org/docs/logos.html	
**Ingress**: https://niantic.helpshift.com/a/ingress/?s=ingress-events&f=brand-and-fan-site-guidelines&p=web	
		
These brands need to have their guidelines updated to these values:
**Scrum Alliance**: https://www.scrumalliance.org/ScrumRedesignDEVSite/media/ScrumAllianceMedia/Files%20and%20PDFs/Infographics/S_BrandGuidelines_2018_rev.pdf
**Bose**: https://developer.bose.com/sites/default/files/Bose%20AR%20Design%20Guidelines%20v1.0.pdf																										
**Roblox**: https://corp.roblox.com/press-kit/																										
**Jamboard**: https://cdn2.hubspot.net/hubfs/159104/ECS/Jamboard/Approved%20Jamboard%20Brand%20Book.pdf																										
**TYPO3**: https://typo3.org/project/brand/style-guide/																										
**Veritas**: https://my.veritas.com/cs/groups/partner/documents/styleguide/mdaw/mdq5/~edisp/tus3cpeapp3855186572.pdf																										
**Nutanix**: https://www.nutanix.com/content/dam/nutanix/en/cmn/documents/nutanix-brandbook.pdf																										
**Internet Explorer**: https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE1qeRY#:~:text=The%20Internet%20Explorer%2011%20logo%20has%20two%20components%3A%20the%20stylized,based%20on%20the%20Segoe%20typeface.
**OCaml**: https://ocaml.org/docs/logos.html																										
**Ingress**: https://niantic.helpshift.com/a/ingress/?s=ingress-events&f=brand-and-fan-site-guidelines&p=web																										

These. brands need to update their hex to these values:
**Pro Tools**: #7926e6																																														

																										

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
skoda

**Issue:** contributes toward #6399

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from [ŠKODA unveils its new logo](https://www.skoda-storyboard.com/en/skoda-world/skoda-unveils-its-new-logo-colours-and-future-identity)
Header_Nove_logo
Source changed to: https://www.skoda-storyboard.com/en/media-room/
Their Company Logo page is returning 404 right now but can be found from the aforementioned link later.
Color: Changed to `#0e3a2f` taken from SVG on their website header.

### CODE REVIEW

**Comment by adamrusted** [comment]
Looks like they've started using the new logo on their site, but similar to the [Dacia](https://github.com/simple-icons/simple-icons/pull/6017) issue, this is probably worth holding fire on until we start seeing the new logo on vehicles. What do other @simple-icons/maintainers think?

**Comment by adamrusted** [comment]
I'm still keeping an eye on this - am yet to see a car showing just the text without the badge...

**Comment by PratyushJayachandran** [comment]
> I'm still keeping an eye on this - am yet to see a car showing just the text without the badge...

I keep checking every week 😂

**Comment by tsinis** [comment]
> > Looks like they've started using the new logo on their site, but similar to the [Dacia](https://github.com/simple-icons/simple-icons/pull/6017) issue, this is probably worth holding fire on until we start seeing the new logo on vehicles. What do other @simple-icons/maintainers think?
> 
> They do show how it will look in front of their cars in the blog post linked but I have not seen third party photos of their cars with the change.

https://youtu.be/ryJsPlA1vbg?t=342

**Comment by adamrusted** [comment]
New logo used on the [https://skoda.co.uk]() website - as good a sign as any for me for us to update it!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10779,8 +10779,8 @@
         },
         {
             "title": "ŠKODA",
-            "hex": "4BA82E",
-            "source": "https://en.wikipedia.org/wiki/File:Skoda_Auto_logo_(2011).svg"
+            "hex": "0E3A2F",
+            "source": "https://www.skoda-storyboard.com/en/media-room/"
```
Comment (adamrusted) [inline _data/simple-icons.json:10783]:
```suggestion
            "source": "https://www.skoda.co.uk"
```

File: icons/skoda.svg
Comment (adamrusted) [inline icons/skoda.svg:1]: I'm seeing some _pretty major_ differences when comparing to the SVG in the new source. Could you re-work this please @PratyushJayachandran? 🙏
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: How about we use skoda.com instead 💀

File: icons/skoda.svg
Comment (PratyushJayachandran) [reply icons/skoda.svg:1]: I have pushed a change. Please review again @adamrusted

**Comment by PratyushJayachandran** [comment]
Updated preview
skoda

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json]: skoda.com isn't one of their websites - we could use skoda-auto.com, but then we'd need to re-compare against that source. I'm happy to leave it as the `.co.uk` for now.

**Review by adamrusted — APPROVED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: I was joking 😛

**Pull request merged**